shared/fs/common/send-in-app-action.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import * as I from 'immutable'
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import {namedConnect} from '../../util/container'
+
+type OwnProps = {
+  path: Types.Path,
+  routePath?: ?I.List<string>,
+  sendIconClassName: string,
+}
+
+const mapDispatchToProps = (dispatch, {path, routePath}: OwnProps) => ({
```

Comment:
why is this connected

Reply:
It needs to dispatch an action. This is populated in the next PR https://github.com/keybase/client/pull/15002